src/main/java/com/afornalik/ox/controller/SkirmishController.java

```diff
@@ -0,0 +1,58 @@
+package com.afornalik.ox.controller;
+
+import com.afornalik.ox.model.board.Board;
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.board.OutOfBoardException;
+import com.afornalik.ox.model.player.PlayerContainer;
+import com.afornalik.ox.view.UIOperations;
+
+class SkirmishController {
+
+    private final UIOperations uiConsole;
+    private final Board board;
+    private final PlayerContainer playerContainer;
+
+    SkirmishController(UIOperations uiConsole, Board board, PlayerContainer playerContainer) {
+        this.uiConsole = uiConsole;
+        this.board = board;
+        this.playerContainer = playerContainer;
+    }
+
+    Board makeMove(FieldStatus fieldStatus) {
+        int index;
+        try {
```

Comment:
Try-catch and inside it next try-catch. This is rather bad practice.
Also, I can see you are using exceptions to validate your input. I don't think it is consider a good practice either. You could just use this checkStatus for both (it could be some enum with status and message for example) or maybe think about some other solution.

Reply:
I erased one try-catch block, and i extracted method.